libero/src/main/groovy/com/connexta/libero/Libero.groovy

```diff
@@ -56,49 +82,93 @@ class Libero {
                     println "Response must be either [y/n]"
                     System.exit(1)
             }
-        }
-        else {
+        } else {
             executeRelease(config)
         }
+        git.checkout(branch: originalBranch)
+    }
+
+    /**
+     * Checks config file for missing values prior to checkout
+     * @param config
```

Comment:
I'd remove those empty `@param` declarations from all the `private` method comments.